lib/typing/columns/columns.go

```diff
@@ -198,7 +194,11 @@ func (c *Columns) GetColumnsToUpdate(uppercaseEscNames bool, args *NameArgs) []s
 			continue
 		}
 
-		cols = append(cols, col.Name(uppercaseEscNames, args))
+		if args == nil {
```

Comment:
This is a stopgap, will look at passing a pointer to `constants.DestinationKind` to this method.

Reply:
`GetColumnsToUpdateRaw` or something?

Reply:
Maybe, I'll look into it in the next PR, I don't want to grow the scope of this PR.